src/components/30-organisms/footer/index.js

```diff
@@ -0,0 +1,285 @@
+// TODO fix that stuff
+/* eslint-disable import/no-extraneous-dependencies */
+import { LitElement, html, svg, css, unsafeCSS } from 'lit-element';
+import { CaretSvg } from '@axa-ch/materials/icons';
+import '@axa-ch/container';
+import defineOnce from '../../../utils/define-once';
+import styles from './index.scss';
+
+import { classMap } from 'lit-html/directives/class-map';
+
+const _listElementHasNoContent = label => {
+  return !label || label.nodeType === 3;
+};
+
+const _extractNestedHref = ev => {
+  const { target } = ev;
+  let element;
+  if (!target || !target.href) {
+    element = ev;
+    while (!element.href) {
+      if (element.target && element.target.parentNode) {
```

Comment:
@raphaellueckl here still not refactored. Please use deconstruction for duplication of code. And we need to clean up element = ev.

Reply:
if you need help, lets do it together

Reply:
Ok I will join you later in SB